airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/JdbcSqlOperations.kt

```diff
@@ -142,7 +142,12 @@ abstract class JdbcSqlOperations : SqlOperations {
                     val uuid = UUID.randomUUID().toString()
 
                     val jsonData = record.serialized
-                    val airbyteMeta = Jsons.serialize(record.record!!.meta)
+                    val airbyteMeta =
+                        if (record.record!!.meta == null) {
```

Comment:
This is pretty ugly. Why should meta ever be null, rather than being the json that serializes into the string below?

Reply:
this is one of the reasons I want to stop passing raw protocol messages deep into our code. Records by default have a null `meta` field, and we don't consistently set a default value anywhere. Ideally we'd define
```kotlin
data SanitizedRecord(
  meta: SanitizedMeta, // non-nullable
  data: JsonNode, // or even just serializedData: String
  namespace: String, // also nonnullable
  originalNamespace: String?, // so that we can match back to state messages, etc.
  name: String,
  // etc
)
```

and convert AirbyteRecord -> SanitizedRecord super early in our call stack. I think @gisripa added something recently that sets us up to do this, but it's not in use yet

(some of this could maybe happen in platform but (a) new connector + old platform = sad, and (b) per-connector logic exists)